jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/rules/ObjectRule.java

```diff
@@ -436,8 +439,8 @@ private void addHashCode(JDefinedClass jclass, JsonNode node) {
                 String propertyName = entry.getKey();
                 JsonNode propertyNode = entry.getValue();
 
-                if (propertyNode.has("excludedFromEqualsAndHashCode") &&
-                        propertyNode.get("excludedFromEqualsAndHashCode").asBoolean()) {
+                if (hasExtensionProperty(propertyNode, "excludedFromEqualsAndHashCode") &&
+                        getExtensionProperty(propertyNode, "excludedFromEqualsAndHashCode").asBoolean()) {
                     filteredFields.remove(ruleFactory.getNameHelper().getPropertyName(propertyName, propertyNode));
```

Comment:
Would we need to handle the getExtensionProperty here?  I could be way off

Reply:
It was worth a deeper look. But in this case no further changes is needed.

The purpose of the "excludedFromEqualsAndHashCode" extension is to specify some _other_ property (or list thereof) that is the name of a field on a generated class* by name in order to exclude said property from consideration in the generated `equals()` and `hashCode()` methods. This line you tagged is processing that other named property (whatever it is) and is thus not related to the goal of this PR. 

\*  (The overall purpose of this class, "ObjectRule.java" is to do with generating classes from `"type": "object"` specs)